### ISSUE TITLE
NVDA does not report the selected status of each character in Microsoft 365

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#11890]
**Please pay attention to my recording presentation and log, Thanks**

### Steps to reproduce:
1. Enter "winword" in the windows + R run dialog to open the Word document.
2. Enter some text, such as: I am a student.
3. Press the Home button to the beginning of the line.
4. Hold down Shift and press Right repeatedly.

### Actual behavior:
NVDA only reports the selected status of some characters.

### Expected behavior:
NVDA should report the selected status of each character

#### NVDA installed/portable/running from source:
installed and portable.

#### NVDA version:
 alpha-21447, NVDA2020.3

#### Windows version:
Windows 10 20H2 (64-bit) build 19042.630

#### Name and version of other software in use when reproducing the issue:
Microsoft 365 MSO (16.0.13426.20270) 64 Bit

#### Other information about your system:
If the following option is selected, the issue cannot be reproduced: NVDA Options ➡ Advanced ➡ Use UI Automation to access Microsoft Word document controls.

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

#### If addons are disabled, is your problem still occuring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes.

[nvda.log](https://github.com/nvaccess/nvda/files/5626374/nvda.log)
[audio](https://update.viyf.org/mc/Record.mp3)
